features/car/access.feature

```diff
@@ -50,14 +50,14 @@ Feature: Car - Restricted access
             | no     |                | yes             |          |                  |                   |      | x     |
             | no     |                | no              | yes      |                  |                   | x    | x     |
             | no     |                | no              | no       |                  | yes               |      | x     |
-            |        | no             |                 |          | no               |                   |      | x     |
-            |        |                | no              |          |                  | no                | x    |       |
+            |        | no             |                 |          | no               |                   |      |       |
```

Comment:
i don't think changes to this test are correct. if you have no access tags, access is implied by a valid way type. for this line in the test we have:

highway=primary (this is set by "routability should be" tests is no highway tag is specified)
access:forward=no
motorcar:forward=no

this means backward access is implied by the way type, while forward access is denied by motorcar:forward (and also by access:forward).

same goes for the other changes to this test.